src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,34 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver vitalii = new Driver(null,"Vitalii", "111511");
+        Driver igor = new Driver(null,"Igor", "201115");
```

Comment:
```suggestion
        Driver vitalii = new Driver("Vitalii", "111511");
        Driver igor = new Driver("Igor", "201115");
```
create one more consructor without id